custom/enikshay/ucr/reports/beneficiary_register_v2.json

```diff
@@ -0,0 +1,751 @@
+                "calculate_total": false,
+                "type": "field",
+                "display": "DST status"
+            },
+            {
+                "sortable": false,
+                "description": null,
+                "format": "default",
+                "transform": {
+                    "type": "translation",
+                    "translations": {
+                        "99dots": "99DOTS",
+                        "contact_centre": "Contact Centre",
+                        "treatment_supporter": "Treatment Supporter",
+                        "merm": "MERM",
+                        "field_officer": "Field Officer"
+                    }
+                },
+                "aggregation": "simple",
+                "field": "adherence_tracking_mechanism",
+                "calculate_total": false,
+                "type": "field",
+                "display": "Adherence Mechanism Assigned"
+            },
+            {
+                "expression": {
+                    "type": "evaluator",
+                    "context_variables": {
+                        "a": {
+                            "datatype": "integer",
+                            "type": "property_name",
+                            "property_name": "total_expected_doses_taken"
+                        },
+                        "b": {
+                            "datatype": "integer",
+                            "type": "property_name",
+                            "property_name": "adherence_total_doses_taken"
+                        }
+                    },
+                    "statement": "str(round(float(b * 100) / a, 2)) + '%'"
+                },
+                "type": "expression",
+                "display": "Total Adherence Score",
+                "column_id": "total_adherence_score"
+            },
+            {
+                "expression": {
+                    "type": "evaluator",
+                    "context_variables": {
+                        "a": {
+                            "datatype": "integer",
+                            "type": "property_name",
+                            "property_name": "doses_per_week"
+                        },
+                        "b": {
+                            "datatype": "date",
+                            "type": "property_name",
+                            "property_name": "adherence_schedule_date_start"
+                        }
+                    },
+                    "statement": "int(round(((today - b).days / 7.0) * a))"
+                },
+                "type": "expression",
+                "display": "total_expected_doses_taken",
+                "column_id": "total_expected_doses_taken"
+            },
+            {
+                "expression": {
+                    "type": "switch",
+                    "switch_on": {
+                        "type": "property_name",
+                        "property_name": "adherence_schedule_id"
```

Comment:
@ctsims do you think this is a reasonable thing to do? @calellowitz has hardcoded the number of days for each adherence schedule for the report that was previously fetched from the lookup table. I don't forsee this changing any time soon, and we don't currently have a mechanism to fetch data from lookup tables in UCR.

Reply:
one alternative i mention below in the other comment is to have the reconciliation task update `doses_per_week` and add that to the data source. then if the fixture changes the reconciliation task will update all the episodes. that would probably back up the queue again but it would be a one time thing